src/identity/manager.rs

```diff
@@ -234,20 +234,24 @@ impl Worker {
         // refresh. In other words, at any point in time, there are no high-priority
         // (not Background) items scheduled to run in the future.
         let mut pending: PriorityQueue<Identity, PendingPriority> = PriorityQueue::new();
-        // The backoff strategy used for retrying operations.
-        let mut cert_backoff: ExponentialBackoff<> = Default::default();
-        // Set the initial values for the backoff. The values are chosen to be reasonable for the CA 
-        // client to be able to recover from transient errors.
-        cert_backoff.initial_interval = Duration::from_millis(500);
-        cert_backoff.current_interval = Duration::from_secs(1);
-        cert_backoff.max_interval = Duration::from_secs(60);
-        // cert_backoff.max_elapsed_time is the maximum elapsed time after instantiating 
-        // [`ExponentialBackoff`](struct.ExponentialBackoff.html) or calling 
-        // [`reset`](trait.Backoff.html#method.reset) after which [`next_backoff`](../trait.Backoff.html#method.reset) 
-        // returns `None`. More info can be found here: https://docs.rs/backoff/0.4.0/backoff.
-        cert_backoff.max_elapsed_time = Some(Duration::from_secs(300));
-        cert_backoff.multiplier = 2.0;
-        cert_backoff.randomization_factor = 0.2;
+        // The backoff strategy used for retrying operations. Sets the initial values for the backoff.
+        // The values are chosen to be reasonable for the CA client to be able to recover from transient
+        // errors.
+        let mut cert_backoff = ExponentialBackoff {
+            initial_interval: Duration::from_millis(500),
+            current_interval: Duration::from_secs(1),
+            max_interval: Duration::from_secs(60),
+            // cert_backoff.max_elapsed_time is the maximum elapsed time after instantiating
+            // [`ExponentialBackoff`](struct.ExponentialBackoff.html) or calling
```

Comment:
Do these links work?

Reply:
No. Just pushed up the FQDNs for those links in the comments. Should be good now.